src/modules/ekf2/EKF/ekf.h

```diff
@@ -373,108 +374,110 @@ class Ekf final : public EstimatorInterface
 	uint8_t _num_bad_flight_yaw_events{0};	///< number of times a bad heading has been detected in flight and required a yaw reset
 	uint64_t _mag_use_not_inhibit_us{0};	///< last system time in usec before magnetometer use was inhibited
-	bool _mag_inhibit_yaw_reset_req{false};	///< true when magnetometer inhibit has been active for long enough to require a yaw reset when conditions improve.
 	float _last_static_yaw{0.0f};		///< last yaw angle recorded when on ground motion checks were passing (rad)
+
+	bool _mag_inhibit_yaw_reset_req{false};	///< true when magnetometer inhibit has been active for long enough to require a yaw reset when conditions improve.
 	bool _mag_yaw_reset_req{false};		///< true when a reset of the yaw using the magnetometer data has been requested
 	bool _mag_decl_cov_reset{false};	///< true after the fuseDeclination() function has been used to modify the earth field covariances after a magnetic field reset event.
 	bool _synthetic_mag_z_active{false};	///< true if we are generating synthetic magnetometer Z measurements
 	bool _non_mag_yaw_aiding_running_prev{false};  ///< true when heading is being fused from other sources that are not the magnetometer (for example EV or GPS).
-
 	bool _is_yaw_fusion_inhibited{false};		///< true when yaw sensor use is being inhibited
 
-	SquareMatrix24f P;	///< state covariance matrix
+	SquareMatrix24f P {};			///< state covariance matrix
 
-	Vector3f _delta_vel_bias_var_accum;		///< kahan summation algorithm accumulator for delta velocity bias variance
-	Vector3f _delta_angle_bias_var_accum;	///< kahan summation algorithm accumulator for delta angle bias variance
+	Vector3f _delta_vel_bias_var_accum {};		///< kahan summation algorithm accumulator for delta velocity bias variance
+	Vector3f _delta_angle_bias_var_accum {};	///< kahan summation algorithm accumulator for delta angle bias variance
 
-	Vector3f _last_vel_obs;			///< last velocity observation (m/s)
-	Vector3f _last_vel_obs_var;		///< last velocity observation variance (m/s)**2
-	Vector2f _last_fail_hvel_innov;		///< last failed horizontal velocity innovation (m/s)**2
-	float _vert_pos_innov_ratio{0.f};	///< vertical position innovation divided by estimated standard deviation of innovation
-	uint64_t _vert_pos_fuse_attempt_time_us{0};	///< last system time in usec vertical position measurement fuson was attempted
-	float _vert_vel_innov_ratio{0.f};		///< standard deviation of vertical velocity innovation
-	uint64_t _vert_vel_fuse_time_us{0};	///< last system time in usec time vertical velocity measurement fuson was attempted
+	Vector3f _last_vel_obs {};			///< last velocity observation (m/s)
+	Vector3f _last_vel_obs_var {};			///< last velocity observation variance (m/s)**2
+	Vector2f _last_fail_hvel_innov {};		///< last failed horizontal velocity innovation (m/s)**2
+	float _vert_pos_innov_ratio;			///< vertical position innovation divided by estimated standard deviation of innovation
```

Comment:
Why was this one removed?

Reply:
Ah, probably just timing relative to https://github.com/PX4/PX4-ECL/commit/ca4932cfc0debacd81dfd1182634d91afca5ecb3

Reply:
```suggestion
	float _vert_pos_innov_ratio{0.f};			///< vertical position innovation divided by estimated standard deviation of innovation
```

Reply:
Ooofffff... Yep, didn't catch that change.  I'll fix it up.